src/fields/fe_value_handler.hh

```diff
@@ -20,12 +20,12 @@
 
 #include "system/index_types.hh"
 #include "fields/field_values.hh"
-#include "fem/mapping_p1.hh"
 #include "fem/finite_element.hh"
 #include "mesh/point.hh"
+#include "la/vector_mpi.hh"
 #include <armadillo>
 
-class VectorMPI;
+class DOFHandlerMultiDim;
```

Comment:
Why these changes in the includes and forward declaration?

Reply:
VectorMPI is no more necessary. DOFHandlerMultiDim was declared via fem/mapping_p1.hh, which is now not included.